app/package.json

```diff
@@ -30,5 +31,12 @@
   "jshintConfig": {
     "esversion": 6,
     "strict": true
+  },
+  "devDependencies": {
+    "eslint": "^3.14.1",
+    "eslint-config-airbnb": "^14.0.0",
+    "eslint-plugin-import": "^2.2.0",
```

Comment:
`eslint`, `eslint-plugin-import`, `eslint-plugin-react`, and `eslint-plugin-jsx-a11y` are required by `eslint-config-airbnb`: https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb#eslint-config-airbnb-1.

But you do have a point, I replaced `eslint-config-airbnb` with `eslint-config-airbnb-base`, that only requires `eslint` and `eslint-plugin-import`.